lib/sidekiq/instrument/middleware/server.rb

```diff
@@ -17,15 +17,21 @@ def call(worker, job, _queue, &block)
       execution_time_ms = (Time.now - start_time) * 1000
       Statter.statsd.measure(metric_name(worker, 'runtime'), execution_time_ms)
       Statter.dogstatsd&.timing('sidekiq.runtime', execution_time_ms, worker_dog_options(worker))
```

Comment:
Could you invert the order here too? Just to be consistent with the rest of the code.